packages/wrangler/src/init.ts

```diff
@@ -243,9 +250,25 @@ export async function initHandler(args: InitArgs) {
 	} else {
 		// Deprecate the `init` command
 		//    if a wrangler.toml file does not exist (C3 expects to scaffold *new* projects)
-		//    and if --from-dash is not set (C3 will run wrangler to communicate with the API)
+		//    and if --from-dash is not set (C3 will run wrangler with --from-dash to communicate with the API)
 		if (!fromDashScriptName) {
-			const replacementC3Command = `\`${packageManager.type} create cloudflare@2\``;
+			const c3Arguments = ["create", "cloudflare@2"];
+
+			if (yesFlag) {
+				c3Arguments.push(
+					".",
+					"--",
+					"--type",
+					"simple",
+					"--ts",
+					"--git",
+					"--no-deploy"
```

Comment:
I have a small concern about this approach.
We are currently pinning the `create-cloudflare` (C3) package to `2`. This means that we could add a new feature C3, bumping the version to `2.1` and this version of Wrangler will use the newer version.
But what happens if that new version contains a feature that requires interaction?
By explicitly listing the arguments here we are really tying this version of Wrangler to C3@2.0.x, right?

I think there are two possible solutions:
- pin this version of Wrangler more tightly to 2.0.x, or
- have some hidden flag on C3 (e.g. `--wrangler-silent`) that C3 (whatever version) can use to ensure that its current features will not make interactive prompts. Making it a hidden arg gives us more flexibility and also avoids confusing the user since many of the web frameworks flows delegate to a 3rd party tool whose interactivity we cannot always control.

Probably the second option is better?

Reply:
This is an interim solution while I add a `-y` or `–wrangler-silent` flag into C3

I agree displaying these extra args will be confusing but it will be temporary and reverted once C3 supports a better approach

Reply:
Implemented `wrangler-defaults` in #3359